rudaux/flows.py

```diff
@@ -155,16 +155,31 @@ def build_snapshot_flows(config, args):
             flows.append(flow)
     return flows
 
+
 @task(checkpoint=False)
 def combine_dictionaries(dicts):
-    return {k : v for d in dicts for k, v in d.items()}
+    return {k: v for d in dicts for k, v in d.items()}
 
-def build_autoext_flows(config, args):
```

Comment:
why get rid of the args input here? I think this might break some other code... and if you're going to get rid of args here, you should get rid of it in all of the `build_*_flows` functions.

If you can't get rid of it in all the build functions, then keep it in all of them.

Reply:
I removed from this function only because it wasn't being used in the function.

Reply:
Actually, I just checked, and it is not used in any of the `build_*__flows`

Reply:
removing it will almost certainly break things. Did you run it to check?

Reply:
No, I still have no machine to run this. 

If it breaks it might be that we're passing the parameter in a function call (and we removed the parameter from the function signature), but this parameter is never being used inside the function, so it should be safe to remove later. 

Anyway, let's keep it for now, but I'll open an issue about this for us to check it later.

Reply:
@Lourenzutti the point I was making is that we have 3 build functions. They should all have the same call signature for consistency / extensibility. If none of them actually use args, then get rid of all of them. If even one of them uses it, keep it in all of them.

Reply:
Got it! 

None of the 3 functions use it. 
I removed `args` from all the three functions and fixed their call in `_flows = build_func(config, args)`